web/templates/pages/tasks.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'base.html' %}
+
+{% block title %}All CodeCatz Tasks{% endblock %}
+
+{% block content%}
+
+  <header class="row">
+    <div class="col-lg-6 col-sm-7">
+      <h1>All CodeCatz Tasks</h1>
+	    <div class="col-lg-6 col-sm-5">
+			<p>This page will show all tasks and who they are assigned to.</p>
+	    </div>
+  </header>
+	  
+	  <div class="row">
+	{% for task in task_list %}
```

Comment:
@ercchy will fix it. Right now i have a headache and am busy being excited this PR doesn't give you an Error page! :P